syntax = "proto3";

package library;

service ReservationService {
  rpc ReserveBook (ReservationRequest) returns (ReservationResponse);
}

message ReservationRequest {
  int32 user_id = 1;
  int32 book_id = 2;
}

message ReservationResponse {
  int32 reservation_id = 1;
}
